uportal-war/src/main/webapp/WEB-INF/flows/edit-account/editLocalAccount.jsp

```diff
@@ -62,44 +62,73 @@
             <!-- Portlet Section -->
             <div class="portlet-section" role="region">
                 <div class="content">
-
                     <table class="portlet-table table table-hover">
```

Comment:
missing a `caption `element into the table.
see [Captions and summaries provide information that can help users find, navigate, and understand tables.](https://www.w3.org/WAI/tutorials/tables/caption-summary/)
if you want the caption only for Screen Reader, using the `.sr-only` class will visually hide the `caption`, and the `.sr-only-focusable` class will ensure that the `caption `becomes visible once focused (for sighted keyboard users).